addons/mail/static/src/chatter/web_portal/chatter.xml

```diff
@@ -73,7 +73,7 @@
             <t t-if="state.composerType">
                 <t t-if="state.composerType === 'message'">
                     <div class="d-flex flex-shrink-0 pt-3 text-truncate small mb-2 ms-5">
-                        <span class="fw-bold">To:</span> <t t-if="toRecipientsText" t-out="toRecipientsText"/>
+                        <span class="fw-bold">To:</span> <div t-if="toRecipientsText" t-out="toRecipientsText" class="ps-1"/>
```

Comment:
Not a huge deal, but usually for inline content we prefer `span` over `div`, just because their default display is respectively `inline-block` and `block`, so this gives a bit more confidence that if layout is broken (flex not working) then at least it's still workable with `block`/`inline-block`